.github/workflows/usersprs.csv

```diff
@@ -1,3 +1,4 @@
 user,pr
 MSevey,3
 liamsi,378
+celestia-bot,378
```

Comment:
@MSevey

Reply:
?? this wasn't happening when I was testing. It should be using the author of the PR.  Will debug.